src/azul/azulclient.py

```diff
@@ -123,22 +123,18 @@ def handle_future(future):
                 nonlocal indexed
                 # @formatter:on
                 # Block until future raises or succeeds
```

Comment:
Wondering if this comment still applies after the removal?

Reply:
removed.